### DISCUSSION TITLE
2023-08-03 CUE Modules Feedback Session / office hours

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#2514] [Community Calls]
As announced on the [2026-06-20 community call](https://github.com/cue-lang/cue/discussions/2441), we are starting fortnightly CUE Modules Feedback Sessions / office hours. See this discussion for context on the modules proposal: https://github.com/cue-lang/cue/discussions/2330

The third session will be on Thursday, **3 Aug, 2023 at 1500 UTC**.

Please reply with any questions/topics you would like to see covered.

### Rough agenda

* Follow-up on previous discussion regarding in-process, configuration-driven "routing" to multiple registries
* Follow-on discussion regarding authentication/authorization against those registries

### Meeting details

https://meet.google.com/wis-hrtw-isf

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar

[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).  

### Recording

https://youtu.be/-6KVm55L4_s

### Slides/materials

https://docs.google.com/presentation/d/1ZOocRkf6C0EFkpnh-AiPB-zqr2fuib2CHWKTNlyadwQ/edit?usp=sharing

### Auto-generated transcript 🆕 

https://docs.google.com/document/d/16cDezJAvFurSD7-bijoGTdb9_P-kRdf39H1GzvaRIOI/edit?usp=sharing

**Comment by verdverm** [comment]
Can confirm, having containers at both `registry.com/foo/bar` and `registry.com/foo/bar/baz` is fine for GCP Artifact Registry, I didn't test container registry, which is backed by a bucket and deprecated anyway, not sure it supports custom manifests to begin with, but not worth trying anyway

**Reply by myitcv** [reply]
Thanks @verdverm. cc @rogpeppe

**Comment by myitcv** [comment]
I've updated the original discussion with links to the recording, slides and transcript.
